src/medkit/medkit-summary.tsx

```diff
@@ -73,16 +67,24 @@ export const MedkitSummary: React.FC = () => {
             <Nav title={'Medical Kit Simulation'} items={navItems}/>
             <div className={'container medkit__content'} data-testid='medkit'>
                 <div className={'row'}>
-                    <div className="col-10">
+                    <div className="col-9">
                         <h1>Medkit Summary</h1>
                     </div>
-                    <div className="col-2">
-                        <button
-                            type={'button'}
-                            onClick={handleDownload}
-                            className={'btn btn-primary d-block ms-auto'}>
-                            Download Summary
-                        </button>
+                    <div className="col-3">
+                        <div className={'d-flex justify-content-end'}>
+                            <button
+                                type={'button'}
+                                onClick={() => window.print()}
+                                className={'btn btn-primary me-2'}>
+                                Print
```

Comment:
Future note: The Print view displays the screen size warning. You'll want to add a print stylesheet that removes this & things like the menu toggle, print & download buttons.